### PR TITLE
cnao: Add new release-0.93 stable branch presubmit

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This PR adds a new presubmit file to the new CNAO stable branch https://github.com/kubevirt/cluster-network-addons-operator/tree/release-0.93
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by RamLavi** [comment]
> Rehearsal jobs created for this PR:

Hey @brianmcarey do you know why didn't prow catch any lanes? It should take the lanes from the stable branch https://github.com/kubevirt/cluster-network-addons-operator/tree/release-0.93

**Comment by RamLavi** [comment]
> > > Rehearsal jobs created for this PR:
> > 
> > 
> > Hey @brianmcarey do you know why didn't prow catch any lanes? It should take the lanes from the stable branch https://github.com/kubevirt/cluster-network-addons-operator/tree/release-0.93
> 
> Hey @RamLavi - are you adding this to the root directory? It needs to be added to the correct job folder for it to be picked up.

oh that's an oops on my part. Thanks man!

**Review by RamLavi — COMMENTED** [review]
/lgtm
/approve

File: github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.93.yaml [hunk 1]
```diff
@@ -0,0 +1,339 @@
+---
+presubmits:
+  kubevirt/cluster-network-addons-operator:
+    - name: pull-e2e-cnao-lifecycle-k8s-release-0.93
+      branches:
+        - release-0.93
+      always_run: true
+      optional: false
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.93.yaml:8]: please consider removing `optional: false` fields
this is the default value

Comment (RamLavi) [reply github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.93.yaml]: [DONE](https://github.com/kubevirt/project-infra/compare/acbc301b21bd2418fc9ec65d66827182448240a2..8f555593c8fb34af66e54e032c8d1135102f3cf1)

**Review by oshoval — APPROVED** [review]

**Pull request merged**